recipes/p_astro/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "p_astro" %}
+{% set version = "0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3df0ab549f07f9de1c9e6b153e4cad65aedaca247d53665a197e78cc8ad80d2a
+  patches:
+    # remove unncessary build dependency
+    - no-pytest-runner.patch
```

Comment:
Let’s remove this patch and simply add pytest-runner to the host section.

Reply:
Done (squashed)